tuf/src/metadata.rs

```diff
@@ -2562,6 +2562,34 @@ mod test {
         assert_eq!(decoded, timestamp);
     }
 
+    // Deserialize timestamp metadata with optional length and hashes
+    #[test]
+    fn serde_timestamp_metadata_optional_length_and_hashes() {
+        let description = MetadataDescription::new(1, None, HashMap::new()).unwrap();
+
+        let timestamp = TimestampMetadataBuilder::from_metadata_description(description)
+            .expires(Utc.ymd(2017, 1, 1).and_hms(0, 0, 0))
+            .build()
+            .unwrap();
+
+        let jsn = json!({
```

Comment:
The name of this test implies to me that we're looking to serialize a length here, but it doesn't appear in this json. Perhaps modify the test name to explicitly say "without", so `serde_timestamp_metadata_without_length_and_hashes`